.env

```
@@ -15,7 +15,7 @@ MYSQL_VERSION=9.6.0
 MYSQL_IMAGE=docker.io/library/mysql:${MYSQL_VERSION}
 POSTGRES_VERSION=18.3
 POSTGRES_IMAGE_VARIANT=-alpine
-MARIADB_VERSION=11.8.6
+MARIADB_VERSION=12.0.2-debian-12-r0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MARIADB_VERSION key should go before the POSTGRES_IMAGE_VARIANT key

(UnorderedKey)